### PR TITLE
GEODE-9628: Documentation fix for setting credentials in client

### PR DESCRIPTION
**Pull request by davebarnes97**
GEODE-9628: Edit format and style

### CODE REVIEW

File: geode-docs/managing/security/implementing_authentication.html.md.erb [hunk 1]
```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
...
 
-## How Authentication Works
+## <a id="authentication-how-it-works"></a>How Authentication Works
 
 When a component initiates a connection to the cluster,
 the `SecurityManager.authenticate` method is invoked.
 The component provides its credentials in the form of properties
 as a parameter to the `authenticate` method.
-The credential is presumed to be the two properties
+The credentials parameter is presumed to be a user/password combination, specified as the two properties
```
Comment (jinmeiliao) [inline geode-docs/managing/security/implementing_authentication.html.md.erb:32]: they don't have to be user/password combinations, they could be tokens, or certificates or username/password combinations.

File: geode-docs/managing/security/implementing_authentication.html.md.erb [hunk 2]
```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
...
-Therefore, the credential of the server become those of the gateway
-sender or receiver.
-
-## How a Client Cache Sets Its Credential
-
-In order to connect with a locator or a server that does authentication,
-a client will need to set its credential, composed of the two properties
+Choose one of two ways to set the server credentials:
+
+- Add settings to the server properties file, or
```
Comment (jinmeiliao) [inline geode-docs/managing/security/implementing_authentication.html.md.erb:47]: change to `Add settings to the server properties file if it's simple username/password combination`

File: geode-docs/managing/security/implementing_authentication.html.md.erb [hunk 3]
```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
...
+the `security-username` and `security-password` properties in whatever way you wish.  For example,
+it might look up values in a database or another external resource.
+
+Gateway senders and receivers communicate as components of their respective server members. Therefore, the
+credentials of the server become those of the gateway sender or receiver.
+
+## <a id="authentication-client-set-creds"></a>How a Client Cache Sets its Credentials
+
+In order to connect with a locator or a server that performs authentication,
+a client must set its credentials, a username and password specified as the two properties
```
Comment (jinmeiliao) [inline geode-docs/managing/security/implementing_authentication.html.md.erb:95]: again, it doesn't have to be username/password, it's whatever property generated by the `security-client-auth-init` class's getCredentials() method call.

File: geode-docs/managing/security/implementing_authentication.html.md.erb [hunk 4]
```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
...
-    to observers of the code.
- 
+For security, take care that credentials set in this manner are not accessible to observers of the code.
 
 
-## How Other Components Set Their Credentials
+## <a id="authentication-component-set-creds"></a>How Other Components Set Their Credentials
 
-`gfsh` prompts for the user name and password upon invocation of
+`gfsh` prompts for the username and password upon invocation of
```
Comment (jinmeiliao) [inline geode-docs/managing/security/implementing_authentication.html.md.erb:147]: add, "and these username/password combination will be provided as properties to the `authenticate` method in the keys of `security-username` and `security-password`".

File: geode-docs/managing/security/implementing_authentication.html.md.erb [hunk 5]
```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
...
 a`gfsh connect` command.
 
-Pulse prompts for the user name and password upon start up.
+Pulse prompts for the username and password upon start up.
```
Comment (jinmeiliao) [inline geode-docs/managing/security/implementing_authentication.html.md.erb:150]: add, "and these username/password combination will be provided as properties to the `authenticate` method in the keys of `security-username` and `security-password`".

**Review by jinmeiliao — CHANGES_REQUESTED** [review]

File: geode-docs/managing/security/implementing_authentication.html.md.erb (hunk 1 above)
Comment (davebarnes97) [reply geode-docs/managing/security/implementing_authentication.html.md.erb]: Thanks, @jinmeiliao Change incorporated.

File: geode-docs/managing/security/implementing_authentication.html.md.erb (hunk 2 above)
Comment (davebarnes97) [reply geode-docs/managing/security/implementing_authentication.html.md.erb]: Thanks, @jinmeiliao Change incorporated.

File: geode-docs/managing/security/implementing_authentication.html.md.erb (hunk 3 above)
Comment (davebarnes97) [reply geode-docs/managing/security/implementing_authentication.html.md.erb]: Thanks, @jinmeiliao Change incorporated.

File: geode-docs/managing/security/implementing_authentication.html.md.erb (hunk 5 above)
Comment (davebarnes97) [reply geode-docs/managing/security/implementing_authentication.html.md.erb:152]: Thanks. Change incorporated.

**Review by jinmeiliao — APPROVED** [review]

**Pull request merged**